src/frontend/shortlist/src/components/layout/profile/UserDetailsInvite.vue

```diff
@@ -0,0 +1,304 @@
+<script>
```

Comment:
Let me set up this page for you as it contains most of my work from the Parent Profile. Let's discuss this whenever you can.

Reply:
We can merge it so you can edit details in develop, I wouldn't recommend change anything in emit though

Reply:
How this component is different than our signup page? Is this not possible to reuse the signup page? It would be great if you can share the video on our slack channel for this PR working flow or briefly explain me. Also, I agree on keeping emit as it is according to your work.

Reply:
Hmm it's just different? I showed it in morning meeting right? Something needs to be taken out. Also the emit needs to be different as backend will run it differently. Backend upon receiving info will find the created accout with email input, then UPDATE info like email and password, that would be the flow, then it would link to set preference

Reply:
All changes does not affect existing structure, I would recommend just merge it, then you can sync and work on it anyway you want, long as the flow can be completed

Reply:
Also another thing would be we will discuss in the next meeting about removing the upload document feature. It is something cannot be fulfilled since we don't have the persona for that anymore. Having it would raise questions that cannot be answered

Reply:
> Hmm it's just different? I showed it in morning meeting right? Something needs to be taken out. Also the emit needs to be different as backend will run it differently. Backend upon receiving info will find the created accout with email input, then UPDATE info like email and password, that would be the flow, then it would link to set preference

I was unable to attend the meeting as I was not feeling well. I hope the scrum master conveyed that message to the team. I would like to request you brief meeting for an explanation of this or a video share on the team channel.

Reply:
Send you on slack dm

Reply:
> All changes does not affect existing structure, I would recommend just merge it, then you can sync and work on it anyway you want, long as the flow can be completed

I am also working on flow completion and am happy to merge it. However, I require some communication before giving approval from my side.